### PR TITLE
#3 project skeleton

### PR DESCRIPTION
**Pull request by Shtramak**
connect to #3

### CODE REVIEW

File: src/main/java/juja/sqlcmd/Application.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package juja.sqlcmd;
```
Comment (ivshapovalov) [inline src/main/java/juja/sqlcmd/Application.java:1]: Please pay attention to package in pom and classes.

**Review by ivshapovalov — DISMISSED** [review]
Fix package name

**Review by kortov — APPROVED** [review]
Посмотри в сторону добавления в pom.xml тегов
<maven.compiler.source> и  <maven.compiler.target>

File: src/test/java/juja/sqlcmd/ApplicationTest.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package juja.sqlcmd;
+
+import org.junit.Test;
+
+public class ApplicationTest {
+    @Test
+    public void testloadApplication(){
```
Comment (AVM82) [inline src/test/java/juja/sqlcmd/ApplicationTest.java:7]: Please pay attention to the open curly brace.
You should add one horizontal whitespace.

**Review by ivshapovalov — APPROVED** [review]

File: src/test/java/juja/sqlcmd/ApplicationTest.java (hunk 2 above)
Comment (Shtramak) [reply src/test/java/juja/sqlcmd/ApplicationTest.java:7]: Sure, I'll fix this point in next PR. Thanks for your remark!

**Review by AVM82 — APPROVED** [review]
LGTM

**Review by DenysMelnychenko — APPROVED** [review]

**Review by adaleorion — APPROVED** [review]
thanks, looks good 👍

**Pull request merged**